driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -296,6 +303,10 @@ public void delete(Object... primaryKey) {
         return new Result<T>(resultSet, mapper, protocolVersion);
     }
```

Comment:
This implies useAlias = true, which breaks the compatibility with previous versions.

This method should set useAlias = false. And I don't think we should expose a variant that allows anything else, because column aliases are not public anyway. The code that needs useAlias = true should call a private method or build the ResultSet directly.